dbms/src/Common/ConfigProcessor.cpp

```diff
@@ -325,7 +325,7 @@ void ConfigProcessor::doIncludesRecursive(
                     return nullptr;
 
                 /// Enclose contents into a fake <from_zk> tag to allow pure text substitutions.
-                zk_document = dom_parser.parseString("<from_zk>" + contents.value() + "</from_zk>");
+                zk_document = dom_parser.parseString("<from_zk>" + *contents + "</from_zk>");
```

Comment:
Why you don't have `value` method?

Reply:
See https://stackoverflow.com/questions/44217316/how-do-i-use-stdoptional-in-c it should be hopefully fixed with the next clang fully supporting `std::optional` from C++17.